### PR TITLE
Adding support for multiple variables of the same type to a plugin for vscode

### PR DESCRIPTION
**Pull request by Nypiaka**

### CODE REVIEW

File: server/test/framework/Server_Tests.cpp [hunk 1]
```diff
@@ -133,6 +134,149 @@ namespace {
                 "buggy_function2");
         }
 
+        static bool comparator(const std::string firstVar, const std::string secondVar, bool mustEqual, bool isInt){
```
Comment (ladisgin) [inline server/test/framework/Server_Tests.cpp:137]: Bad naming

Comment (ladisgin) [reply server/test/framework/Server_Tests.cpp:137]: Bad naming of parameter

File: server/test/framework/Server_Tests.cpp [hunk 2]
```diff
@@ -133,6 +134,83 @@ namespace {
                 "buggy_function2");
         }
 
+        static bool checkEquals(const tests::Tests::TestCaseParamValue& first, const tests::Tests::TestCaseParamValue& second){
+            return std::stoi(first.view->getEntryValue(nullptr)) == std::stoi(second.view->getEntryValue(nullptr));
+        }
+
+        void checkDifferentVariablesTrue_C(BaseTestGen &testGen){
```
Comment (ladisgin) [inline server/test/framework/Server_Tests.cpp:141]: unnecessary extract to function

File: server/test/framework/Server_Tests.cpp [hunk 3]
```diff
@@ -133,6 +134,83 @@ namespace {
...
+            checkDifferentVariables_C(testGen, true);
+        }
+
+        void checkDifferentVariablesFalse_C(BaseTestGen &testGen){
```
Comment (ladisgin) [inline server/test/framework/Server_Tests.cpp:145]: unnecessary extract to function

**Review by ladisgin — CHANGES_REQUESTED** [review]

File: server/test/framework/Server_Tests.cpp (hunk 2 above)
Comment (Nypiaka) [reply server/test/framework/Server_Tests.cpp]: done

**Review by ladisgin — APPROVED** [review]

**Pull request merged**